environments/common/inventory/group_vars/all/basic_users.yml

```diff
@@ -3,5 +3,10 @@
 # See: ansible/roles/basic_users/README.md 
 # for variable definitions.
 
+ondemand_user_password: "{{ vault_openondemand_default_user_password }}"
+
 basic_users_homedir: /home
-basic_users_users: []
+basic_users_users:
+  - name: ood_user
```

Comment:
Don't like this name, it's not just for OOD. I would call it "demo_user" I think, makes it clear its not supposed to be for production. And the stackhpc environment should be modified to use this too, rather than the current test user.